[2024-09-03T07:53:17.086+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-03T07:53:17.099+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_ingestion.fetch_weather manual__2024-09-03T07:53:15.084758+00:00 [queued]>
[2024-09-03T07:53:17.104+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_ingestion.fetch_weather manual__2024-09-03T07:53:15.084758+00:00 [queued]>
[2024-09-03T07:53:17.105+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-09-03T07:53:17.115+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): fetch_weather> on 2024-09-03 07:53:15.084758+00:00
[2024-09-03T07:53:17.121+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1521) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-03T07:53:17.122+0000] {standard_task_runner.py:72} INFO - Started process 1523 to run task
[2024-09-03T07:53:17.123+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_data_ingestion', 'fetch_weather', 'manual__2024-09-03T07:53:15.084758+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag2.py', '--cfg-path', '/tmp/tmp4jugpqj5']
[2024-09-03T07:53:17.123+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask fetch_weather
[2024-09-03T07:53:17.158+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_data_ingestion.fetch_weather manual__2024-09-03T07:53:15.084758+00:00 [running]> on host 670a75e9450f
[2024-09-03T07:53:17.219+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_ingestion' AIRFLOW_CTX_TASK_ID='fetch_weather' AIRFLOW_CTX_EXECUTION_DATE='2024-09-03T07:53:15.084758+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-03T07:53:15.084758+00:00'
[2024-09-03T07:53:17.220+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-03T07:53:17.220+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-03T07:53:17.221+0000] {logging_mixin.py:190} INFO - Current task name:fetch_weather state:running start_date:2024-09-03 07:53:17.099823+00:00
[2024-09-03T07:53:17.221+0000] {logging_mixin.py:190} INFO - Dag name:weather_data_ingestion and current dag run status:running
[2024-09-03T07:53:17.221+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-03T07:53:17.377+0000] {logging_mixin.py:190} INFO - Error: Unable to retrieve data. Status code: 400
[2024-09-03T07:53:17.378+0000] {python.py:240} INFO - Done. Returned value was: []
[2024-09-03T07:53:17.393+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-03T07:53:17.394+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_data_ingestion, task_id=fetch_weather, run_id=manual__2024-09-03T07:53:15.084758+00:00, execution_date=20240903T075315, start_date=20240903T075317, end_date=20240903T075317
[2024-09-03T07:53:17.400+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-09-03T07:53:17.401+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-09-03T07:53:17.401+0000] {logging_mixin.py:190} INFO - Dag name:weather_data_ingestion queued_at:2024-09-03 07:53:15.091440+00:00
[2024-09-03T07:53:17.401+0000] {logging_mixin.py:190} INFO - Task hostname:670a75e9450f operator:PythonOperator
[2024-09-03T07:53:17.416+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-03T07:53:17.432+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-03T07:53:17.434+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
